lib/ably/version.rb

```diff
@@ -1,6 +1,6 @@
 module Ably
   VERSION = '1.2.3'
-  PROTOCOL_VERSION = '1.2'
+  PROTOCOL_VERSION = '2.0'
```

Comment:
We're dropping from Decimal to Integer service versions. See https://github.com/ably/specification/pull/119